### DISCUSSION TITLE
developer-case-studies/transcriber/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#222] [Website Discussions]
# developer-case-studies/transcriber/

A Developers Story

https://fcp.cafe/developer-case-studies/transcriber/

**Comment by latenitefilms** [comment]
This is an INCREDIBLE case study! Thanks for sharing!

**Comment by florian-duffe** [comment]
Thanks, Alex! What an effort! And what an App! It's very elegant!
I already could put Transcriber to good use with a 2 hour (only Audio!) interview, which will hopefully be published on this website, soon. 
3 things I recognised:
- My M1 Max gets very hot and loses battery fast, even when Transcriber is in the background.
- When deleting a search: the text jumps to a random place in the transcript. I would prefer it to stay at the last place. Usually you want to listen on from the place of last search.
- with this 2 hour file, performance goes down (e. g. clicking on pause takes a few seconds). I tested with shorter files - everything snappy.

**Comment by shylock74** [comment]
Hi Florian,

I'll try to answer as completely as possible.

> My M1 Max gets very hot and loses battery fast, even when Transcriber is in the background.

I know that.
I'm working on a different technology, it will be unveiled in the next week, because at the moment I'm working on some edge case on this current release. 🙁

> When deleting a search: the text jumps to a random place in the transcript. I would prefer it to stay at the last place. Usually you want to listen on from the place of last search.

Ok, never tried this.
Consider it done ASAP.

> With this 2 hour file, performance goes down (e. g. clicking on pause takes a few seconds). I tested with shorter files - everything snappy.

I know that too.
I'm workin' on this and I hope to release something faster soon.

Thanks, really thanks for the feedbacks.
BTW: sayin "It's very elegant!" to a developer means a lot, it's one of the most beautiful compliment you can veer say! ❤️

Alex
